rest/service_management.go

```diff
@@ -227,6 +228,26 @@ func (s *ManagementService) MarkManyComplete(rw http.ResponseWriter, r *http.Req
 	gimlet.WriteJSON(rw, struct {
 		Message string `json:"message"`
 	}{
-		Message: "mark jobs complete by type successful",
+		Message: "mark jobs complete by filter successful",
+	})
+}
+
+// MarkCompleteByPrefix is an http.Handlerfunc marks all jobs with the
+// specified prefix and status complete.
+func (s *ManagementService) MarkCompleteByPrefix(rw http.ResponseWriter, r *http.Request) {
+	vars := gimlet.GetVars(r)
+	prefix := vars["prefix"]
+	filter := vars["filter"]
+
+	ctx := r.Context()
+	if err := s.manager.CompleteJobsByPrefix(ctx, management.StatusFilter(filter), prefix); err != nil {
+		gimlet.WriteResponse(rw, gimlet.MakeTextErrorResponder(errors.Wrapf(err,
```

Comment:
Does it really make sense to wrap here?  Presumably we know where this error is coming from, and the thing we actually care about is the stack that we already put in `err`. If we wrap here, there will be two stacks in the error message, and I don't think we'll care about the one from this function.

Reply:
I think this is more of a Sam thing. I am not super opinionated on this, but we have been wrapping everything else.